modules/cron4s/src/main/scala/pureconfig/module/cron4s/package.scala

```diff
@@ -0,0 +1,15 @@
+package pureconfig.module
+
+import _root_.cron4s.expr.CronExpr
+import _root_.cron4s.Cron
+import pureconfig.ConfigReader
+import pureconfig.error.CannotConvert
+
+package object cron4s {
+
+  implicit val cronExprReader: ConfigReader[CronExpr] =
+    ConfigReader.fromString(str =>
+      Cron.parse(str).fold(
+        err ⇒ Left(CannotConvert(str, "CronExpr", err.getMessage)),
```

Comment:
Nitpick: for consistency, can you please write the two arrows here using ASCII characters, as you wrote above?